test/liquid_argon_reduced.jl

```diff
@@ -0,0 +1,59 @@
+include("../src/nbody_simulation.jl")
```

Comment:
why do you need to include a src file here?

Reply:
Usually, I do the following steps: start a new console, start Julia, change the working directory, run this script using include("liquid_argon_reduced.jl") and it pulls the necessary for simulation code calling include("../src/nbody_simulation.jl"). Probably this is an awkward way to test the code and there is a better approach)

Reply:
Just setup Revise.jl and do `using DiffEqPhysics`, and then whenever you change the package it will automatically update the function calls.